example/src/App.tsx

```diff
@@ -265,8 +266,18 @@ class App extends React.Component<any, any> {
       const result = await web3.eth.sign(hash, address)
 
       // verify signature
-      const signer = recoverPublicKey(result, hash)
-      const verified = signer.toLowerCase() === address.toLowerCase()
+      let signer = null
+      let verified = false
+      const bytecode = await web3.eth.getCode(address)
+      const isContract = !/^(0x)?0+$/.test(bytecode)
```

Comment:
(nit) Isn't easier to just test if length is greater than 2?

Reply:
Depending on version of web3, it'll return `0x`, `0x0`, or `0x00`